test/features/resource_freshness.feature

```diff
@@ -0,0 +1,46 @@
+@schema_metadata
+Feature: Resource freshness
+
+Scenario Outline: An editor, admin or sysadmin user, when I go to the dataset new page, the field field-next_update_due should not be visible with the correct values
+        Given "<User>" as the persona
+        When I log in
+        And I go to "/dataset/new"
+        Then I shold not see an element with id "field-next_update_due"
+
+
+        Examples: Users
+        | User              |
+        | Admin             |
+        | TestOrgAdmin      |
+        | TestOrgEditor     |
+
+
+    Scenario Outline: An editor, admin or sysadmin user, when I go to the dataset new page, the field field-next_update_due should be visible with the correct values
```

Comment:
Update the title to match test.
`when the correct update frequency is selected the field field-next_update_due should be visible`?